commercetools-models/src/main/java/io/sphere/sdk/shippingmethods/queries/ShippingMethodsByOrderEditGet.java

```diff
@@ -0,0 +1,24 @@
+package io.sphere.sdk.shippingmethods.queries;
+
+import com.neovisionaries.i18n.CountryCode;
+import io.sphere.sdk.orderedits.OrderEdit;
+import io.sphere.sdk.queries.MetaModelGetDsl;
+import io.sphere.sdk.shippingmethods.ShippingMethod;
+import io.sphere.sdk.shippingmethods.expansion.ShippingMethodExpansionModel;
+
+import java.util.List;
+
+/**
+ * Retrieves all the shipping methods that can ship to the given orderEdit.
+ *
+ * {@include.example io.sphere.sdk.shippingmethods.queries.ShippingMethodsByOrderEditGetIntegrationTest#execution()}
+ */
+public interface ShippingMethodsByOrderEditGet extends MetaModelGetDsl<List<ShippingMethod>, ShippingMethod, ShippingMethodsByCartGet, ShippingMethodExpansionModel<ShippingMethod>> {
+    static ShippingMethodsByOrderEditGet of(final OrderEdit orderEdit, final CountryCode countryCode) {
```

Comment:
Better use something like

`Identifiable<OrderEdit>`

which allows to use References as well as OrderEdits to be used to query the shipping methods